userdocs/profiles.dev/docs/author-docs/profile-definition-schema.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 7
+---
+
+# Profile definition schema
+
+<!-- TODO: autogen this in like yaml or smth -->
```

Comment:
I've seen this done elsewhere.. I think maybe check the operator-sdk docs?

Reply:
yeh on my list to do later